lte/protos/enodebd.proto

```diff
@@ -117,6 +137,9 @@ service Enodebd {
   // Reboot every connected eNodeB
   rpc RebootAll (magma.orc8r.Void) returns (magma.orc8r.Void);
 
+  // Download file to eNodeB
+  rpc Download (DownloadRequest) returns (DownloadResponse);
```

Comment:
Considering that this is specifically for upgrading eNB firmware, that context should be included in the documentation here.